test/runtests.jl

```diff
@@ -458,83 +458,6 @@ end
     @test _x509_certificate.issuer_name == x509_certificate.issuer_name
 end
 
-# https://www.openssl.org/docs/man3.0/man7/OSSL_PROVIDER-legacy.html
-@testset "Encrypt" begin
-    if OpenSSL.version_number() ≥ v"3"
-        OpenSSL.load_legacy_provider()
-    end
-    evp_ciphers = [
-        EvpEncNull(),
-        EvpBlowFishCBC(), # legacy
-        EvpBlowFishECB(), # legacy
-        #EvpBlowFishCFB(), // not supported
-        EvpBlowFishOFB(), # legacy
-        EvpAES128CBC(),
-        EvpAES128ECB(),
-        #EvpAES128CFB(), // not supported
-        EvpAES128OFB(),
-    ]
-
-    foreach(evp_ciphers) do evp_cipher
-        sym_key = random_bytes(evp_cipher.key_length)
-        init_vector = random_bytes(evp_cipher.init_vector_length)
-
-        enc_evp_cipher_ctx = EvpCipherContext()
-        encrypt_init(enc_evp_cipher_ctx, evp_cipher, sym_key, init_vector)
-
-        dec_evp_cipher_ctx = EvpCipherContext()
-        decrypt_init(dec_evp_cipher_ctx, evp_cipher, sym_key, init_vector)
-
-        in_string = "OpenSSL Julia"
-        in_data = IOBuffer(in_string)
-        enc_data = IOBuffer()
-
-        cipher(enc_evp_cipher_ctx, in_data, enc_data)
-        seek(enc_data, 0)
-        @show String(read(enc_data))
-        seek(enc_data, 0)
-
-        dec_data = IOBuffer()
-        cipher(dec_evp_cipher_ctx, enc_data, dec_data)
-        out_data = take!(dec_data)
-        out_string = String(out_data)
-
-        @test in_string == out_string
-    end
-end
-
-@testset "EncryptCustomKey" begin
-    # EvpBlowFishECB is legacy, consider using EvpAES128ECB instead
```

Comment:
Instead of being removed, this test could be moved to a non-legacy cipher as suggested in the comment.